### ISSUE TITLE
[bug] After upgrading to version 11.0.0, the `include` option does not work on windows

### ISSUE THREAD

**Issue by panmenghan** [silverwind/precompress#10]
After upgrading to version 11.0.0:
```console
$ precompress -i 'js' src
Error: No matching files found
   at Tt (file:///C:/Users/.../node_modules/.pnpm/precompress@11.0.0/node_modules/precompress/bin/precompress.js:19:1337)
```

It might be caused by these changes of rrdir:
https://github.com/silverwind/rrdir/compare/10.1.2...10.1.3

Install the two versions of rrdir:
```console
$ pnpm add rrdir1012@npm:rrdir@10.1.2
$ pnpm add rrdir1013@npm:rrdir@10.1.3
```

Run `rrdir`:
```js
import {rrdir as rrdir_old} from 'rrdir1012'
import {rrdir} from 'rrdir1013'

for await (const entry of rrdir_old('src', {
  include: ['**/*.js'],
})) {
  console.log('old', entry)
  break
}

for await (const entry of rrdir_old('src', {
  include: ['**\\*.js'],
})) {
  console.log('old with windows sep', entry)
  break
}

for await (const entry of rrdir('src', {
  include: ['**/*.js'],
})) {
  console.log('new', entry)
  break
}

for await (const entry of rrdir('src', {
  include: ['**\\*.js'],
})) {
  console.log('new with windows sep', entry)
  break
}
```

Output:
```
old { path: 'src\\a.js', directory: false, symlink: false }
old with windows sep { path: 'src\\a.js', directory: false, symlink: false }
new with windows sep { path: 'src\\a.js', directory: false, symlink: false }
```

On **windows** with linux sep, the new version will not print the result.

**Comment by silverwind** [comment]
Thanks for the report. I think I should add Windows to CI on both repos to catch these. It does seem to work fine on OSs with `/` path separator.

**Comment by silverwind** [comment]
Should be fixed with 11.0.1, please try it.

**Comment by panmenghan** [comment]
Thanks, it works!
